packages/suite-desktop/src/modules/coinjoin.ts

```diff
@@ -61,20 +61,9 @@ export const init: Module = ({ mainWindow, store }) => {
             await backend.run({ ...settings, torSettings: store.getTorSettings() });
             backends.push(backend);
 
-            backend.on('interceptor', (event: InterceptedEvent) => {
-                if (event.type === 'INTERCEPTED_REQUEST') {
-                    logger.debug('request-interceptor', `${event.method} - ${event.details}`);
-                }
-                if (event.type === 'INTERCEPTED_RESPONSE') {
-                    logger.debug(
-                        'request-interceptor',
-                        `request to ${event.host} took ${event.time}ms and responded with status code ${event.statusCode}`,
-                    );
-                }
-                if (event.type === 'NETWORK_MISBEHAVING') {
-                    logger.debug('request-interceptor', 'networks is misbehaving');
-                }
-            });
+            backend.on('interceptor', (event: InterceptedEvent) =>
+                ipcMain.emit('module/request-interceptor', new Event(event.type), event),
```

Comment:
```suggestion
                ipcMain.emit('module/request-interceptor', event),
```
new Event(event.type), (electron.Event) should be added automatically ?

Reply:
instead of overloading electon.ipcMain i will create another standalone `EventEmitter` just for that purpose